guide/07-working-with-bigdata/02-Making-your-data-accessible-to-GA-server.ipynb

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
<p>I think there is no such item with title "FileShareFolder" in the item list below. Please recheck.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/804/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>